lib/order.rb

```diff
@@ -0,0 +1,13 @@
+class Order
+  attr_reader :order_list
+  def initialize
+    @order_list = []
+  end 
+
+  def order_dish(dish_name, quantity = 1)
+    for _ in 0...quantity
```

Comment:
Could use the .times method here, this would reduce the number of lines and make it clear about what you're trying to accomplish with these lines of code.